src/patternfly/components/Button/examples/buttonStates.hbs

```diff
@@ -1,3 +1,58 @@
-{{#> button btnClass="pf-is-primary pf-has-focus"}}Focused{{/button}}
-{{#> button btnClass="pf-is-primary pf-is-active" btnAriaPressed="true"}}Active{{/button}}
-{{#> button btnClass="pf-is-primary pf-is-disabled" btnTabIndex="-1" btnAriaDisabled="true"}}Disabled{{/button}}
\ No newline at end of file
+<div class="pf-l-level">
```

Comment:
Why is this layout here? there should be no layout information in the component example code.

Reply:
That's a good question - I wanted to lay the example out nicely because it's not one, it's many buttons. Is it a rule that you can't do that? Is it a rule that you don't include more than one "thing" per example? If so, buttons is going to be super long. Or I could do something in the middle and put each type with all its states in one example.